tasks/check-for-sessions.yml

```diff
@@ -0,0 +1,18 @@
+---
+  - name: Check for running meetings
+    uri:
+      url: "https://{{ bbb_hostname }}/bigbluebutton/api/getMeetings?checksum={{ ('getMeetings' + bbb_secret) | hash('sha1') }}"
+      return_content: yes
+    register: bbb_getMeetings
+    delegate_to: localhost
```

Comment:
Is there a reason this is run on localhost and not on the target host? I would prefer that this task is also run on the target, for situations where the controller can't directly access the host

Reply:
I don't know why I chose to run this on the controller - I just changed that.

Another issue seems to be that "geerlingguy.docker" and "geerlingguy.nodejs" run before this check gets executed. I don't yet know why or where these dependency roles are started...